service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/IssuedOrderAdminNotificationHandler.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.hmcts.reform.fpl.handlers;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.fpl.enums.IssuedOrderType;
+import uk.gov.hmcts.reform.fpl.model.event.EventData;
+import uk.gov.hmcts.reform.fpl.service.email.NotificationService;
+import uk.gov.hmcts.reform.fpl.service.email.content.OrderIssuedEmailContentProvider;
+
+import java.util.Map;
+
+import static uk.gov.hmcts.reform.fpl.NotifyTemplates.ORDER_ISSUED_NOTIFICATION_TEMPLATE_FOR_ADMIN;
+
+@Component
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class IssuedOrderAdminNotificationHandler {
+    private final NotificationService notificationService;
+    private final HmctsAdminNotificationHandler adminNotificationHandler;
+    private final OrderIssuedEmailContentProvider orderIssuedEmailContentProvider;
+
+    public void sendOrderIssuedNotificationToAdmin(final EventData eventData,
```

Comment:
Name again could just be sendToAdmin as we already know from the class that we are dealing with issued order notifications